src/reference/architecture/architecture.rst

```diff
@@ -0,0 +1,45 @@
+.. _architecture.components:
+
+Cylc 8 Components
+=================
+
+The main Cylc 8 system components are:
+
+Cylc Scheduler
+--------------
+
+   - The workflow engine core, Python 3 based
+   - Includes the **CLI** (Command Line Interface)
+   - And **TUI**, a new Terminal UI application
```

Comment:
These lines shouldn't be indented.

The extra indentation will put the bullets inside a blockquote.